### ISSUE TITLE
Move from Travis CI to Circle CI

### ISSUE THREAD

**Issue by carodew** [18F/a11y-metrics#33]
@toolness can you fill in some more context details?

**Comment by toolness** [comment]
Sure!

Basically, this dashboard heavily relies on Travis CI's [cron jobs](https://docs.travis-ci.com/user/cron-jobs/) feature to periodically re-generate the metrics dashboard. 18F is moving away from Travis to Circle CI sometime very soon, but Circle CI doesn't *yet* have this feature. However, [they are planning on implementing it soon](https://discuss.circleci.com/t/scheduled-builds/5038/6), dubbing the feature "scheduled builds", and Aidan [reached out](https://gsa-tts.slack.com/archives/C1GTZ1DFZ/p1503599438000356) to make sure we are on their early access roll-out.

So in other words, this is blocked on Circle CI actually adding support for scheduled builds. Once they've done that, we can proceed with this. Depending on when they do that and when we move away from Travis, this might mean the a11y dashboard is out-of-service for a short period of time.

**Comment by hollyallen** [comment]
FYI: This repository has been using Travis CI, however access to Travis CI is now turning off.

**Comment by maya** [comment]
+1, it would be great to get this automatically running again, so we can stay up to date on how TTS projects are doing a11y-wise. Since CircleCI now supports cron jobs, looks like this can move forward 🎉
